### ISSUE TITLE
Runtime errors on Linux

### ISSUE THREAD

**Issue by krzyk** [astog/MoreLenses#34]
I tried using the mod on Linux and I get a bunch of erros in Lua.log:
```
InGame: Loading InGame UI - C:/Emu/AppDataParent/Sid Meier's Civilization VI/Mods/MoreLenses/Lenses/ModLens_CityOverlap
ModLens_CityOverlap: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ModLens_CityOverlap:           City Overlap Panel
ModLens_CityOverlap: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Runtime Error: C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_CityOverlap.lua:182: operator + is not supported for nil + number
stack traceback:
    C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_CityOverlap.lua:182: in function 'OnReoffsetPanel'
    C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_CityOverlap.lua:279: in function 'Initialize'
    C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_CityOverlap.lua:306: in function '(main chunk)'
    [C]: in function '(anonymous)'^M
Lua callstack:^M
Error loading file where=, file=C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_CityOverlap.lua
InGame: Loading InGame UI - C:/Emu/AppDataParent/Sid Meier's Civilization VI/Mods/MoreLenses/Lenses/ModLens_Resource
ModLens_Resource: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ModLens_Resource:            Resource Panel
ModLens_Resource: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Runtime Error: C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_Resource.lua:355: operator + is not supported for nil + number
stack traceback:
    C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_Resource.lua:355: in function 'OnReoffsetPanel'
    C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_Resource.lua:424: in function 'Initialize'
    C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_Resource.lua:448: in function '(main chunk)'
    [C]: in function '(anonymous)'^M
Lua callstack:^M
Error loading file where=, file=C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Lenses\ModLens_Resource.lua
```

This is the first time I look at the logs, maybe it is uppercase/lowercase related? None of the lens buttons works right now for me in the game.

EDIT:
And also:
```
Error opening/reading where=C:\Emu\AppAssets\DLC\expansion2\UI\Replacements\MinimapPanel_Expansion1.lua:6: , file=C:\Emu\AppDataParent\Sid Meier's Civilization VI\Mods\MoreLenses\Base\Assets\UI\minimappanel.lua
```

**Comment by feelotraveller** [comment]
Was also having problems on Linux.

The changes in the linked commit fixed the issues for me. :)

Thanks.

**Comment by anders-larsson** [comment]
Hi,

I'm also having problems with More Lenses. When I load it (no other workshop mods enabled) everything in the lower left corner disappears. That is, the map and all available options.

This is with Rise and Fall and Gathering Storm. In Lua.log I see the following error if it helps:
```
Error opening/reading where=C:\Emu\AppAssets\DLC\expansion2\UI\Replacements\MinimapPanel_Expansion1.lua:6: , file=\home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Base\Assets\UI\minimappanel.lua
Error opening/reading where=C:\Emu\AppAssets\DLC\expansion2\UI\Replacements\ModalLensPanel_Expansion1.lua:6: , file=\home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Base\Assets\UI\Panels\modallenspanel.lua
MapPinPopup: initializing MapTacks.IconOptions(0)
MapPinPopup: Selected 13 columns
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/DedicationPopup
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/DisloyalCityChooser
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/EraProgressPanel
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/EraReviewPopup
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/ExpansionIntro
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/GovernorAssignmentChooser
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/GovernorPanel
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/HistoricMoments
InGame: Loading InGame UI - ../../../DLC/expansion2/UI/Additions/WorldCrisisPopup
InGame: Loading InGame UI - /home/user/.local/share/Steam/steamapps/workshop/content/289070/871712879/Lenses/ModLens_CityOverlap
ModLens_CityOverlap: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ModLens_CityOverlap:           City Overlap Panel
ModLens_CityOverlap: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Runtime Error: \home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_CityOverlap.lua:182: operator + is not supported for nil + number
stack traceback:
        \home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_CityOverlap.lua:182: in function 'OnReoffsetPanel'
        \home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_CityOverlap.lua:279: in function 'Initialize'
        \home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_CityOverlap.lua:306: in function '(main chunk)'
        [C]: in function '(anonymous)'
Lua callstack:
Error loading file where=, file=\home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_CityOverlap.lua
InGame: Loading InGame UI - /home/user/.local/share/Steam/steamapps/workshop/content/289070/871712879/Lenses/ModLens_Resource
ModLens_Resource: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ModLens_Resource:            Resource Panel
ModLens_Resource: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Runtime Error: \home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_Resource.lua:355: operator + is not supported for nil + number
stack traceback:
        \home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_Resource.lua:355: in function 'OnReoffsetPanel'
        \home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_Resource.lua:424: in function 'Initialize'
        \home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_Resource.lua:448: in function '(main chunk)'
        [C]: in function '(anonymous)'
Lua callstack:
Error loading file where=, file=\home\user\.local\share\Steam\steamapps\workshop\content\289070\871712879\Lenses\ModLens_Resource.lua
```

**Comment by astog** [comment]
Merged the changes #35 .
Thanks again @krzyk for linux issue fix.
